pmg/templates/committee_detail.html

```diff
@@ -38,35 +32,41 @@ <h3 class="collapse-link"><span class="fa fa-caret-right"></span> <a href="#">Ab
             <div class="form-group">
               <select class="form-control">
                 <option data-target="#cm" data-toggle="tab" value="#cm">Committee meetings</option>
-                {% if committee.calls_for_comments %}
                 <option data-target="#cc" data-toggle="tab" value="#cc">Calls for comment</option>
-                {% endif %}
-                {% if committee.tabled_committee_reports %}
                 <option data-target="#tr" data-toggle="tab" value="#tr">Tabled reports</option>
-                {% endif %}
-                {% if recent_questions %}
                 <option data-target="#qr" data-toggle="tab" value="#qr">Questions/replies</option>
-                {% endif %}
               </select>
             </div>
           </nav>
           <div class="tab-pane fade in active" id="cm">
             <div class="cte-meetings-filter-outer hidden-xs">
               <div class="btn-group cte-meetings-filter-inner">
+                {% if filtered_meetings['six-months'] | length > 0 %}
```

Comment:
The first one to be shown needs to have the 'active' class on the button and the tab-pane.